src/CorsairLightingProtocolTinyUSBHID.cpp

```diff
@@ -26,7 +26,7 @@ uint8_t const hid_report[] = {HID_USAGE_PAGE_N (HID_USAGE_PAGE_VENDOR | 0xC0, 2)
 							  HID_COLLECTION_END};
 /* clang-format on */
 
-Adafruit_USBD_HID TinyUSBDevice_HID(hid_report, sizeof(hid_report), HID_ITF_PROTOCOL_NONE, 2, true);
+Adafruit_USBD_HID tudHid(hid_report, sizeof(hid_report), HID_ITF_PROTOCOL_NONE, 2, true);
```

Comment:
did you test with 200 and more leds? why did you choose `2` here?

Reply:
Good catch! I was doing some testing and forgot to change it back.

Reply:
can we set it to `0`?

Reply:
I haven't tried but I don't think we need to. 25 FPS is the max iCue allows and we get that with 1.